lib/LINE/Bot/API/Builder/SendMessage.pm

```diff
@@ -23,6 +23,7 @@ sub add_text {
     $self->add(+{
         type => 'text',
         text => $args{text},
+        sender => $args{sender},
```

Comment:
We probably want to avoid sending `"sender": null` to Message server. Let's checks whether `$args{sender}` is defined first before we pass it thru.